### ISSUE TITLE
Booster creation exits program with no error. ( _LIB.LGBM_BoosterCreate )

### ISSUE THREAD

**Issue by rabyj** [microsoft/LightGBM#5464]
I made a [bug report on Optuna](https://github.com/optuna/optuna/issues/3960) that could be from a problem further down the line, here.

**TL;DR**

Codes goes up to [_LIB.LGBM_BoosterCreate](https://github.com/microsoft/LightGBM/blob/d78b6bc2fdf96c87e3cb61f2d497a962e3270c91/python-package/lightgbm/basic.py#L2730) through [cvbooster = Booster(tparam, train_set)](https://github.com/microsoft/LightGBM/blob/master/python-package/lightgbm/engine.py#L485) with what seems to be the correct training data. It just stops after printing some info LightGBM info messages.

With reduced dataset (542 samples), code goes past cvbooster = Booster(tparam, train_set)) the first time, but it stops the second time it gets to _LIB.LGBM_BoosterCreate (creation of internal training set).

Here is the **complete copy** of the issue:

### Expected behavior

I'm putting two problems together because they have the same input, just one version has a smaller input. I imagine they are related. This might be a LightGBM problem, and not optuna. The issue can be closed if that is the case.

Expected behavior : [ LightGBMTunerCV](https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_cv.py) begins trials after LightGBM prints total bin number.

Observed behaviour:  LightGBMTunerCV.run() ends and exits the program without any trials or errors. Last prints are related to total bin size. Internal Training dataset not constructed.

After reducing the total data from 1096 samples (~130 mb) to 542 samples, the problem **slightly** changes. (same data source/classes, still same number of samples for each class). I get a bit more prints indicating the training set was constructed, but then it stops similarly.

If the dataset is reduced even further, trials seem to start.

### Environment

- Optuna version: 3.0.0
- Python version: 3.8.10
- OS: Linux-5.15.0-46-generic-x86_64-with-glibc2.29 (Ubuntu 20.04.5 LTS)
- lightgbm : 3.3.2

### Error messages, stack traces, or logs

SITUATION 1
Logs with verbosity 4:
```shell
[I 2022-09-01 17:18:25,811] A new study created in memory with name: no-name-f7558179-1685-4799-a668-4d05bb21188a
[LightGBM] [Debug] Dataset::GetMultiBinFromAllFeatures: sparse rate 0.000000
[LightGBM] [Debug] init for col-wise cost 0.026579 seconds, init for row-wise cost 2.166356 seconds
[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 4.252611 seconds.
You can set `force_col_wise=true` to remove the overhead.
[LightGBM] [Info] Total Bins 7731833
```

Full stack context before end.
```
__init__ (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/lightgbm/basic.py:2610)
_make_n_folds (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/lightgbm/engine.py:410)
cv (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/lightgbm/engine.py:605)
__call__ (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py:319)
_run_trial (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py:196)
_optimize_sequential (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py:160)
_optimize (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py:66)
optimize (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/study.py:419)
_tune_params (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py:653)
tune_feature_fraction (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py:570)
run (/home/local/USHERBROOKE/rabj2301/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py:545)
tune_lbgm (/home/local/USHERBROOKE/rabj2301/Projects/sources/epi_ml/epi_ml/python/core/estimators.py:179)
main (/home/local/USHERBROOKE/rabj2301/Projects/sources/epi_ml/epi_ml/python/other_estimators.py:132)
<module> (/home/local/USHERBROOKE/rabj2301/Projects/sources/epi_ml/epi_ml/python/other_estimators.py:170)
_run_code (/usr/lib/python3.8/runpy.py:87)
_run_module_as_main (/usr/lib/python3.8/runpy.py:194)
```

SITUATION 2

Logs
```
[I 2022-09-02 16:09:15,257] A new study created in memory with name: no-name-73c934d6-7610-4800-bb86-995160cc7deb
[LightGBM] [Debug] Dataset::GetMultiBinFromAllFeatures: sparse rate 0.000000
[LightGBM] [Debug] init for col-wise cost 0.001908 seconds, init for row-wise cost 0.575387 seconds
[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 1.194562 seconds.
You can set `force_col_wise=true` to remove the overhead.
[LightGBM] [Info] Total Bins 5493186
[LightGBM] [Info] Number of data points in the train set: 274, number of used features: 30321
[LightGBM] [Debug] Dataset::GetMultiBinFromAllFeatures: sparse rate 0.000000
[LightGBM] [Debug] init for col-wise cost 0.001943 seconds, init for row-wise cost 0.570320 seconds
[LightGBM] [Warning] Auto-choosing col-wise multi-threading, the overhead of testing was 1.317071 seconds.
You can set `force_col_wise=true` to remove the overhead.
[LightGBM] [Info] Total Bins 5493186
```

The full stack context before the end is the same in this case, no error traceback.

### Steps to reproduce

I was not able to create a minimal working example, since reducing the dataset size even more makes the problem disappear. 

Nonetheless, I pickled my input data (542 samples) to permit reproducing the problem with the following code (when the input is in the same folder):

```python
import lzma
import pathlib
import pickle

import optuna.integration.lightgbm as lgb

params = {
    "objective": "multiclass",
    "num_class": 4,
    "metric": "multi_logloss",
    "boosting_type": "dart",
    "seed": 42,
    "verbosity": 4,
}

with lzma.open(pathlib.Path(__file__).parent.resolve() / "input.pickle.xz", "rb") as f:
    input_data = pickle.load(f)

dtrain = input_data["dtrain"]
folds = input_data["folds"]

tuner = lgb.LightGBMTunerCV(
    params,
    dtrain,
    folds=folds,
    show_progress_bar=False,
)


tuner.run()
```
[Link to data](https://filedropper.com/d/s/tPWm52taJw0PYDnMvtIJ1sGeocJeaL)

### Additional context (optional)

essential part of my own code, which is not that far from [lightgbm_tuner_cv.py example](https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_cv.py):
```python
    params = {
        "objective": "multiclass",
        "num_class": len(ea_handler.classes),
        "metric": "multi_logloss",
        "boosting_type": "dart",
        "seed": 42,
        "verbosity": 4,
    }

    total_data = ea_handler.create_total_data()
    folds = list(ea_handler.split(total_data))
    dtrain = lgb.Dataset(total_data.signals, label=total_data.encoded_label)

    tuner = lgb.LightGBMTunerCV(
        params,
        dtrain,
        folds=folds,
        show_progress_bar=False,
    )

    tuner.run()
```

- Data is multiclass
- lgbm dataset was created with:
- `total_data.signals`: `ndarray[num_features, n_samples] (dtype=float32)`
- `total_data.encoded_label`: `ndarray[n_samples] (dtype=int64)`
- `folds` is a list of the modified output of StratifiedKFold (repeated values for oversampling, but still same domain and format), so train and test idx. k=2 in kfold.
- `free_raw_data=False` changes nothing

([github](https://github.com/rabyj/epi_ml/commit/8960351c37e5fde20735f840413f899201d9b24f#diff-e370a91fe6e07469d65071226d2066a1c8b3883f87f7c534876d9fecd8429b01R137))

Codes goes up to [`_LIB.LGBM_BoosterCreate`](https://github.com/microsoft/LightGBM/blob/d78b6bc2fdf96c87e3cb61f2d497a962e3270c91/python-package/lightgbm/basic.py#L2730) through [`cvbooster = Booster(tparam, train_set)`](https://github.com/microsoft/LightGBM/blob/master/python-package/lightgbm/engine.py#L485) with what seems to be the correct training data. It just stops after printing some info LightGBM info messages.

With reduced dataset (542 samples), code goes past `cvbooster = Booster(tparam, train_set)`) the first time, but it stops the second time it gets to `_LIB.LGBM_BoosterCreate` (creation of internal training set).

**Comment by jameslamb** [comment]
Thanks very much for the report!

Since you've created an example that reproduces the error in your environment, could you try upgrading to the latest `lightgbm` built from source and see if you still experience problems?

See https://github.com/microsoft/LightGBM/tree/master/python-package#install-from-github for instructions on how to do that.

**Comment by rabyj** [comment]
Building failed with something that went far above my head, for now.

```shell
(venv-epilap-pytorch) rabyj:~/.../epilap/$ pip install --force-reinstall --no-binary :all: lightgbm
Collecting lightgbm
  Downloading lightgbm-3.3.2.tar.gz (1.5 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.5/1.5 MB 10.0 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting wheel
  Downloading wheel-0.37.1.tar.gz (66 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 66.4/66.4 kB 10.6 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
Collecting numpy
  Downloading numpy-1.23.2.tar.gz (10.7 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 10.7/10.7 MB 10.5 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting scipy
  Downloading scipy-1.9.1.tar.gz (42.0 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 42.0/42.0 MB 8.7 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error
  
  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [60 lines of output]
      The Meson build system
      Version: 0.62.2
      Source dir: /tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964
      Build dir: /tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-mknbz64m/build
      Build type: native build
      Project name: SciPy
      Project version: 1.9.1
      C compiler for the host machine: cc (gcc 9.4.0 "cc (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0")
      C linker for the host machine: cc ld.bfd 2.34
      C++ compiler for the host machine: c++ (gcc 9.4.0 "c++ (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0")
      C++ linker for the host machine: c++ ld.bfd 2.34
      Host machine cpu family: x86_64
      Host machine cpu: x86_64
      Compiler for C supports arguments -Wno-unused-but-set-variable: YES
      Library m found: YES
      Fortran compiler for the host machine: gfortran (gcc 9.4.0 "GNU Fortran (Ubuntu 9.4.0-1ubuntu1~20.04.1) 9.4.0")
      Fortran linker for the host machine: gfortran ld.bfd 2.34
      Program cython found: YES (/tmp/pip-build-env-aaxqoq5a/overlay/bin/cython)
      Program pythran found: YES (/tmp/pip-build-env-aaxqoq5a/overlay/bin/pythran)
      Program cp found: YES (/usr/bin/cp)
      Program python3 found: YES (/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/bin/python)
      Found pkg-config: /usr/bin/pkg-config (0.29.1)
      Library npymath found: YES
      Library npyrandom found: NO
      Found CMake: /usr/local/bin/cmake (3.22.0)
      Run-time dependency openblas found: NO (tried pkgconfig and cmake)
      Run-time dependency openblas found: NO (tried pkgconfig and cmake)
      
      ../../scipy/meson.build:130:0: ERROR: Dependency "OpenBLAS" not found, tried pkgconfig and cmake
      
      A full log can be found at /tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-mknbz64m/build/meson-logs/meson-log.txt
      + meson setup --native-file=/tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-native-file.ini -Ddebug=false -Doptimization=2 --prefix=/usr /tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964 /tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-mknbz64m/build
      Traceback (most recent call last):
        File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
          main()
        File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
          json_out['return_val'] = hook(**hook_input['kwargs'])
        File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 130, in get_requires_for_build_wheel
          return hook(config_settings)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 923, in get_requires_for_build_wheel
          with _project(config_settings) as project:
        File "/usr/lib/python3.8/contextlib.py", line 113, in __enter__
          return next(self.gen)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 902, in _project
          with Project.with_temp_working_dir(
        File "/usr/lib/python3.8/contextlib.py", line 113, in __enter__
          return next(self.gen)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 547, in with_temp_working_dir
          yield cls(source_dir, tmpdir, build_dir)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 463, in __init__
          self._configure(reconfigure=bool(build_dir) and not native_file_mismatch)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 494, in _configure
          self._meson(
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 477, in _meson
          return self._proc('meson', *args)
        File "/tmp/pip-build-env-aaxqoq5a/overlay/lib/python3.8/site-packages/mesonpy/__init__.py", line 472, in _proc
          subprocess.check_call(list(args))
        File "/usr/lib/python3.8/subprocess.py", line 364, in check_call
          raise CalledProcessError(retcode, cmd)
      subprocess.CalledProcessError: Command '['meson', 'setup', '--native-file=/tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-native-file.ini', '-Ddebug=false', '-Doptimization=2', '--prefix=/usr', '/tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964', '/tmp/pip-install-7gho42yy/scipy_966f46848c194dfda6d6aad9f0da5964/.mesonpy-mknbz64m/build']' returned non-zero exit status 1.
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.
```

**Comment by rabyj** [comment]
Did `sudo apt-get install libopenblas-dev` and launched build again, we'll see how that goes.

**Comment by rabyj** [comment]
For reference, building via `pip install --no-binary :all: lightgbm` did not solve the problem. (same Environment
info). 

Once installing through github source, I got a new problem, probably because optuna 3.0.0 doesn't support unreleased versions of LightGBM (past 3.3.2).

Env info:
- Python version: 3.8.10
- OS: Linux-5.15.0-46-generic-x86_64-with-glibc2.29 (Ubuntu 20.04.5 LTS)
- **lightgbm: 3.3.2.99**
- Optuna: 3.0.0
- scipy:1.8.1


Log/Traceback
```shell
[I 2022-09-06 13:05:13,200] A new study created in memory with name: no-name-7767ac86-e9a2-4aad-9c85-22d6beb83947
[W 2022-09-06 13:05:14,946] Trial 0 failed because of the following error: TypeError("cv() got an unexpected keyword argument 'fobj'")
Traceback (most recent call last):
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py", line 196, in _run_trial
    value_or_values = func(trial)
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py", line 319, in __call__
    cv_results = lgb.cv(self.lgbm_params, train_set, **self.lgbm_kwargs)
TypeError: cv() got an unexpected keyword argument 'fobj'
Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/__main__.py", line 39, in <module>
    cli.main()
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 430, in main
    run()
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher/../../debugpy/../debugpy/server/cli.py", line 284, in run_file
    runpy.run_path(target, run_name="__main__")
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 321, in run_path
    return _run_module_code(code, init_globals, run_name,
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 135, in _run_module_code
    _run_code(code, mod_globals, init_globals,
  File "/home/local/USHERBROOKE/<user>/.vscode/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_runpy.py", line 124, in _run_code
    exec(code, run_globals)
  File "/home/local/USHERBROOKE/<user>/Projects/sources/epi_ml/epi_ml/python/other_estimators.py", line 170, in <module>
    main(sys.argv[1:])
  File "/home/local/USHERBROOKE/<user>/Projects/sources/epi_ml/epi_ml/python/other_estimators.py", line 132, in main
    estimators.tune_lbgm(ea_handler, cli.logdir)
  File "/home/local/USHERBROOKE/<user>/Projects/sources/epi_ml/epi_ml/python/core/estimators.py", line 190, in tune_lbgm
    tuner.run()
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py", line 545, in run
    self.tune_feature_fraction()
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py", line 570, in tune_feature_fraction
    self._tune_params([param_name], len(param_values), sampler, "feature_fraction")
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py", line 653, in _tune_params
    study.optimize(
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/study.py", line 419, in optimize
    _optimize(
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py", line 66, in _optimize
    _optimize_sequential(
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py", line 160, in _optimize_sequential
    frozen_trial = _run_trial(study, func, catch)
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py", line 234, in _run_trial
    raise func_err
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/study/_optimize.py", line 196, in _run_trial
    value_or_values = func(trial)
  File "/home/local/USHERBROOKE/<user>/Projects/epilap/venv-epilap-pytorch/lib/python3.8/site-packages/optuna/integration/_lightgbm_tuner/optimize.py", line 319, in __call__
    cv_results = lgb.cv(self.lgbm_params, train_set, **self.lgbm_kwargs)
TypeError: cv() got an unexpected keyword argument 'fobj'
```

Said `lgbm_kwargs` object contains the following
```python
{'num_boost_round': 1000, 'fobj': None, 'feval': None, 'feature_name': 'auto', 'categorical_feature': 'auto', 'early_stopping_rounds': None, 'verbose_eval': None, 'callbacks': None, 'folds': [(...), (...)], 'nfold': 5, 'stratified': True, 'shuffle': True, 'show_stdv': True, 'seed': 0, ...}
```

**Comment by rabyj** [comment]
I've compiled my basic observations from trying different combinations of versions here: https://github.com/optuna/optuna/issues/3960#issuecomment-1238483244

The same problem arises with both latest dev versions (for optuna and LightGBM). @jameslamb

**Comment by rabyj** [comment]
optuna/optuna#3960 was closed since this seems to be a problem arising within LightGBM. 

Since I'm unable to create a minimal code example without sharing 60mb+ of data, but can reproduce the bug with **lightgbm 3.3.2.99**, are there any additional logs I am unaware of that I could provide on the inner workings of LightGBM during the execution of _LIB.LGBM_BoosterCreate?

Or else, somewhere I could host the unpickled data? (it comes from HDF5 files)
